buddyboss/buddyboss-platform
# Redirect to discussion page after create new discussion

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2307 .

### How to test the changes in this Pull Request:

1. Create a group and set the privacy to hidden
2. Associate a forum
3. Add a non-admin user to the group as a member
4. Login/View as a group member of the hidden group you just created
5. Try to create a discussion
6. Notice the issue


### Proof Screenshots or Video

https://www.loom.com/share/149f3d5497824b80a0f1170c515cc6ab

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
